packages/programs/src/programs/DefaultProgram.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { IBaseWorld, WorldConsumer } from "@latticexyz/world-consumer/src/experimental/WorldConsumer.sol";
+
+import { EntityId } from "@dust/world/src/EntityId.sol";
+import { Player } from "@dust/world/src/codegen/tables/Player.sol";
+
+import { IAttachProgramHook, IDetachProgramHook } from "@dust/world/src/ProgramInterfaces.sol";
+
+import { AllowedCaller } from "../codegen/tables/AllowedCaller.sol";
+import { Owner } from "../codegen/tables/Owner.sol";
+
+import { SmartItem } from "../codegen/tables/SmartItem.sol";
+import { UniqueEntity } from "../codegen/tables/UniqueEntity.sol";
+
+contract DefaultProgram is IAttachProgramHook, IDetachProgramHook, WorldConsumer {
+  constructor(IBaseWorld _world) WorldConsumer(_world) { }
+
+  function _requireOwner(EntityId target) internal view returns (bytes32) {
+    bytes32 smartItemId = SmartItem.get(target);
```

Comment:
nit: I would pass the smartItemId to this function instead of getting it here, so the return value can be removed, makes the requireOwner function more focused on just doing what it is supposed to do